change/@fluentui-react-breadcrumb-preview-8e37ea24-6643-4dca-a824-7919c7e0581c.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "feat: breadcrumb - removed appearance prop",
```

we need to make sure changelog contains explicit breaking change. because tooling we use for releases is limited in terms of DX simple `BREAKING CHANGE:` is all we can do.

```suggestion
  "comment": "BREAKING CHANGE: remove `appearance` prop and set  `current` prop  for BreadcrumbButton last item",
```